### PR TITLE
Small Updates

### PR DESCRIPTION
**Pull request by Cahllagerfeld**

### CODE REVIEW

File: src/components/artifacts/artifact-node-sheet/DetailCards.tsx [hunk 1]
```diff
@@ -14,95 +19,131 @@ import {
...
-								className="inline-flex items-center gap-0.5"
-								rounded={false}
-								emphasis="subtle"
-							>
-								{stepData?.body?.status === "running" ? (
-									<Spinner className="mr-1 h-4 w-4 border-[2px]" />
+							<>
+								{stepDetail.data ? (
+									<Tag
+										color={getExecutionStatusTagColor(pipelineRun.data.body?.status)}
```
Comment (Cahllagerfeld) [inline src/components/artifacts/artifact-node-sheet/DetailCards.tsx:112]: This needs to be double checked

Comment (Cahllagerfeld) [reply src/components/artifacts/artifact-node-sheet/DetailCards.tsx:112]: @schustmi If i recall correctly thats the part, where we just link a step, but dont need the pipeline status for it right?

**Review by atzin-escandia — APPROVED** [review]

File: src/components/artifacts/artifact-node-sheet/DetailCards.tsx (hunk 1 above)
Comment (schustmi) [reply src/components/artifacts/artifact-node-sheet/DetailCards.tsx:112]: Yes, the step run should not be coupled to the pipeline run

Comment (schustmi) [reply src/components/artifacts/artifact-node-sheet/DetailCards.tsx:112]: just make it green always, there is no way the artifact exists without the producer step having finished, at least in the current dag visualizer implementation

**Pull request merged**